### PR TITLE
Feat/guardrails

### PR DESCRIPTION
**Pull request by alexa-perlov**

### CODE REVIEW

**Review by scottschreckengaust — APPROVED** [review]

**Review by scottschreckengaust — APPROVED** [review]

**Review by MichaelWalker-git — APPROVED** [review]
LGTM

**Review by MichaelWalker-git — APPROVED** [review]

File: examples/agent-with-guardrails/main.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#####################################################################################
+# Terraform module examples are meant to show an _example_ on how to use a module
+# per use-case. The code below should not be copied directly but referenced in order
+# to build your own root module that invokes this module
+#####################################################################################
+variable "region" {
```
Comment (prabirsekhri) [inline examples/agent-with-guardrails/main.tf:6]: any reason why the region variable is in main.tf instead of variables.tf?

**Review by prabirsekhri — APPROVED** [review]
LGTM, approved

**Pull request merged**